test/hubspot/record/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"os/signal"
+	"strconv"
+	"syscall"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/providers/hubspot"
+	connTest "github.com/amp-labs/connectors/test/hubspot"
+	"github.com/amp-labs/connectors/test/utils"
+	"github.com/brianvoe/gofakeit/v6"
+)
+
+const samplePropertyChange = `{
+  "appId": 4210286,
+  "eventId": 100,
+  "subscriptionId": 2902227,
+  "portalId": 44237313,
+  "occurredAt": 1730750483646,
+  "subscriptionType": "contact.propertyChange",
+  "attemptNumber": 0,
+  "objectId": 74999542704,
+  "changeSource": "CRM",
+  "propertyName": "message",
+  "propertyValue": "sample-value"
+}`
+
+func main() {
+	// Handle Ctrl-C gracefully.
+	ctx, done := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
+	defer done()
+
+	// Set up slog logging.
+	utils.SetupLogging()
+
+	// Get the Hubspot connector.
+	conn := connTest.GetHubspotConnector(ctx)
+	defer utils.Close(conn)
+
+	// Write an artificial contact to Hubspot.
+	writeresult, err := conn.Write(ctx, common.WriteParams{
+		ObjectName: "contacts",
+		RecordId:   "",
+		RecordData: map[string]any{
+			"email":     gofakeit.Email(),
+			"phone":     gofakeit.Phone(),
+			"company":   gofakeit.Company(),
+			"website":   gofakeit.URL(),
+			"lastname":  gofakeit.LastName(),
+			"firstname": gofakeit.FirstName(),
+		},
+	})
+	if err != nil {
+		utils.Fail("error writing to hubspot", "error", err)
+	}
+
+	propMsg := hubspot.WebhookMessage{}
+
+	if err := json.Unmarshal([]byte(samplePropertyChange), &propMsg); err != nil {
+		utils.Fail("error unmarshalling property change message", "error", err)
+	}
+
+	recordId, err := strconv.Atoi(writeresult.RecordId)
```

Comment:
I see that Identifier is converted from string to int to later inside `GetWebhookResultFromWebhookMessage` to be converted back to streing. Should we drop the int format?